test/unit/sqs/error-handlers.test.ts

```diff
@@ -1,15 +1,14 @@
 import statsd from "../../../src/config/statsd";
-import {jiraOctokitErrorHandler} from "../../../src/sqs/error-handlers";
-import {Context} from "../../../src/sqs/index";
+import {jiraOctokitErrorHandler, webhookMetricWrapper} from "../../../src/sqs/error-handlers";
+import {Context, ErrorHandlingResult} from "../../../src/sqs/index";
 import {getLogger} from "../../../src/config/logger";
 import {JiraClientError} from "../../../src/jira/client/axios";
 import {RateLimitingError as OldRateLimitingError} from "../../../src/config/enhance-octokit";
 import {Octokit} from "probot";
 import {RateLimitingError} from "../../../src/github/client/errors";
 import {AxiosError} from "axios";
 
-
-describe("jiraOktokitErrorHandler", () => {
+describe("error-handlers", () => {
```

Comment:
Sorry for too many line changes in these file. I had to add a top-level describe, because not we test 2 things from that file, each of them having its own "describe"